CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Rename `compare_version` to `get_max_version` for readability
 
+- Add user metadata recipe
+
```

Comment:
You also need to update `coreDriverInterfaceSupported.json` file. Please see the node PR.